eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageTypes.scala

```diff
@@ -171,6 +171,51 @@ case class AcceptChannel(temporaryChannelId: ByteVector32,
   val channelType_opt: Option[ChannelType] = tlvStream.get[ChannelTlv.ChannelTypeTlv].map(_.channelType)
 }
 
+// NB: this message is named is open_channel2 in the specification.
```

Comment:
nit: remove second `is` ->

// NB: this message is named open_channel2 in the specification.